tensorboard/plugins/core/core_plugin.py

```diff
@@ -451,17 +450,19 @@ def define_flags(self, parser):
 
   def fix_flags(self, flags):
     """Fixes standard TensorBoard CLI flags to parser."""
+    FlagsException = base_plugin.FlagsException
     if flags.inspect:
       if flags.logdir and flags.event_file:
-        raise ValueError(
+        raise FlagsException(
             'Must specify either --logdir or --event_file, but not both.')
       if not (flags.logdir or flags.event_file):
-        raise ValueError('Must specify either --logdir or --event_file.')
+        raise FlagsException('Must specify either --logdir or --event_file.')
     elif not flags.db and not flags.logdir:
-      raise ValueError('A logdir or db must be specified. '
-                       'For example `tensorboard --logdir mylogdir` '
-                       'or `tensorboard --db sqlite:~/.tensorboard.db`. '
-                       'Run `tensorboard --helpfull` for details and examples.')
+      raise FlagsException('A logdir or db must be specified.\n\n'
```

Comment:
I’d prefer to omit the change to this plugin’s particular error message
from this PR, which is otherwise a straightforward refactoring. Are you
open to moving this message change into a separate PR?

Reply:
Sure.